contracts/stake-cw20-reward-distributor/src/contract.rs

```diff
@@ -144,6 +144,10 @@ fn get_distribution_msg(deps: Deps, env: &Env) -> Result<Option<CosmosMsg>, Cont
     let config = CONFIG.load(deps.storage)?;
     let last_payment_block = LAST_PAYMENT_BLOCK.load(deps.storage)?;
     let block_diff = env.block.height - last_payment_block;
+    if block_diff <= 0 {
+        return Ok(None);
```

Comment:
The audit report recommendation: `We recommend reverting with an error when block_diff is zero.`

Shouldn't we return an error here?

Reply:
I'm unsure on erroring here due to the conflicts it could cause when handling config updates

Reply:
An error here actually seems like it would be nice. Something along the lines of "rewards have already been distributed for this block".